docs/source/user/notebook.rst

```diff
@@ -160,18 +160,11 @@ You can iterate through the kernel history in a document cell using ``Alt Up-Arr
 Cell Toolbar
 ^^^^^^^^^^^^
 
-If there is enough room for it, each cell has a toolbar that provides quick access to
-commonly-used functions. If you would like to disable the cell toolbar, run:
+In each cell, there is a toolbar that provides quick access to
+commonly-used functions, if there is enough room to display it on the first line.
 
-.. code:: bash
-
-    jupyter labextension disable @jupyterlab/cell-toolbar-extension
-
-on the command line. You can enable it again by running:
-
-.. code:: bash
-
-    jupyter labextension enable @jupyterlab/cell-toolbar-extension
+If you would like to disable the cell toolbar, open the Settings Editor, choose the
+Cell Toolbar in the left panel, and uncheck "Show cell toolbar".
```

Comment:
I think keeping the CLI instructions as "for administrators" makes sense. It is certainly easier to use them when deploying JupyterLab to JupyterHub or binder that the settings editor route.